cineast-core/src/main/java/org/vitrivr/cineast/core/util/ocr/HungarianAlgorithm.java

```diff
@@ -1,4 +1,4 @@
-package org.vitrivr.cineast.core.util;
+package org.vitrivr.cineast.core.util.ocr;
```

Comment:
This is not OCR-specific functionality, it's also used for the skeleton stuff for example.